### ISSUE TITLE
Раскрытие услуг на Тернополь и другие регионы области

### ISSUE THREAD

**Issue by NizhnikAlex** [e-government-ua/iBP#376]
Тернопіль:
1. Дозвіл на викиди забруднюючих речовин в атмосферне повітря стаціонарними джерелами 

Тернопільський р-н:
1. Виправлення технічної помилки у відомостях з державного земельного кадастру, яка була допущена органом, що здійснює його ведення
2. Внесення до Державного земельного кадастру відомостей (змін до них) про земельну ділянку

Заліщики:
1. Надання відомостей з Державного земельного кадастру у формі витягу з Державного земельного кадастру про обмеження у використанні земель
2. Надання містобудівних умов та обмежень забудови земельної ділянки

Чортків: 
1. Надання містобудівних умов та обмежень забудови земельної ділянки
2. Надання довідки про наявність у Державному земельному кадастрі відомостей про одержання у власність земельної ділянки у межах норм безоплатної приватизації за певним видом її цільового призначення (використання)
Відповідальний Шимків Артур Володимирович tsnap_mista_chertkov@ukr.net логін/пароль на тестовий cnap_chortT_1

**Comment by NizhnikAlex** [comment]
Тернополь раскрыта https://test.igov.org.ua/service/137/general
заказ услуги только по ЭЦП
Обработка - новый пользователь с именем и паролем Tern_test
Также доступна для аккаунтов 	
Ірина Кельнер, Ольга Ванчура, Ванчура Ольга Леонівна

**Comment by NizhnikAlex** [comment]
Тернопільський район:
https://test.igov.org.ua/service/41/general
https://test.igov.org.ua/service/99/general

Обработка - новый пользователь с именем и паролем Tern_test
Также доступна для аккаунтов Ірина Кельнер, Ванчура Ольга Леонівна

Послуги ідентичні, для розкриття потрібна буде шапка заяви (в цих заявах ПІБ керівника не вносили - щось вони часто змінювались) і мейл відповідального.

**Comment by kelnerira** [comment]
@NizhnikAlex  Тернополь раскрыта https://test.igov.org.ua/service/137/general
Ця послуга у Тернополі надається через міський ЦНАП,  екологія погодила 2-ге коло обробки через портал. Відповідальна у Департаменті екології та природніх ресурсів облдержадміністрації  Трояновська Світлана Михайлівна	eco_ter@tr.ukrtel.net	0988709018 Створіть будь ласка для неї тестовий кабінет, також потрібно замість надсилання на електронку додати 2-ге коло обробки на порталі.

**Comment by NizhnikAlex** [comment]
@kelnerira Так і є - через міський ЦНАП обласного центру реалізовано для всієї області.
Уточнення по крокам.
Наразі реалізовано такий механізм:
1. Заявник подає заяву
2. ЦНАП проводить попередню перевірку, якщо все ок, у Департамент екології надходить лист з даними. Після обробки даних Департамент передає документи до ЦНАП.
3. ЦНАП опрацьовує друге коло звернення
4. Заявник забирає документи.

Ви хочете додати ще одне коло для екологів (між першим і другим ЦНАПа), або на другому колі дати доступ не ЦНАПу, а екологам?

**Comment by kelnerira** [comment]
@NizhnikAlex хочу додати ще 1 коло обробки для екологів замість надсилання на мейл.

**Comment by kelnerira** [comment]
Тернопільський р-н дані відповідальних https://docs.google.com/document/d/1nzQEUSXCTBUDH2qdtHEISyFjIgR-yp3GT76J8JaEe7A/edit?usp=sharing

**Comment by NizhnikAlex** [comment]
Так, тоді давайте далі з'ясуємо.

Заліщики і Чортків - вони кадастровими послугами також без ЦНАП займаються?

"Надання містобудівних умов та обмежень забудови земельної ділянки" в мене реалізація ЦНАП-Архітектура-ЦНАП. Як у вас повинно бути?

Річ у тім, що розкрити я ці послуги можу, але якщо треба їх змінювати - це вже буде значно довше.

**Comment by kelnerira** [comment]
Ні, Заліщики і Чортків - заявлені послуги надають через ЦНАП.  
Заліщикам також потрібна послуга " Надання відомостей з державного земельного кадастру у формі витягу з державного земельного кадастру про землі в межах території адміністративно-територіальних одиниць" 
 Відповідальна у ЦНАП Заліщицької РДА  Волошин Ірина Михайлівна	zalcnap@ukr.net	роб.2-28-38,   
Та сама послуга є у заявці ЦНАП Теребовлянського р-ну, відповідальна Качанюк Ольга Михайлівна	Cnap.trb@ukr.net	03512 10 29,          03512 11 31 
Тернопільський район частину послуг земельників надає через ЦНАП, частину - самостійно. 
Послуги, які надаються самостійно,  потрібно буде змінювати. Ми Вам для зручності зробимо таблицю з вказанням де саме які зміни потрібно внести.

**Comment by NizhnikAlex** [comment]
Логін/пароль для екологів EcoTern
Послугу трохи змінив. Зараз трохи доопрацюю, десь за пів години можна буде подивитись

**Comment by NizhnikAlex** [comment]
Містобудівні умови розкрив на Чортків і Заліщицький район.
Для Чорткова пароль chortT_rai_cnap1
Для Заліщицького району zalisch_rai_cnap1

Процес опрацьовується у 3 кроки ЦНАП-Архітектура-ЦНАП.

Наразі користувачі з тестовими логінами під'єднані як до груп ЦНАП, так і до груп архітектури. На бойовому сервері ЦНАП буде мати доступ тільки до своїх кроків, архітектори - до своїх.

**Comment by kelnerira** [comment]
тут зауваження за результатами тестування виправлення тех.помилки https://docs.google.com/spreadsheets/d/1Wxk2BYCZ01hC03WPj4LiifSB5DtWdfb1bDoaphJ9ywA/edit?usp=sharing

**Comment by kelnerira** [comment]
https://docs.google.com/spreadsheets/d/1It852JH-p7F6TFe5RwPIF6QPQf8CWBtjcPelxcGgfxs/edit?usp=sharing тут будуть зміни за результатами тестування у екології, поки що подивіться на зауваження щодо ЕЦП

**Comment by sofiadanylevych** [comment]
По містобудівних умовах та обмеженнях заявки Для Заліщицького району zalisch_rai_cnap1
Не приходять на цей логін

**Comment by kelnerira** [comment]
@NizhnikAlex
1. послуги Екології, друге коло обробки.
Відсутня кнопка перевірки ЕЦП. Передбачається, що ЕЦП перевіряє лише ЦНАП? А екологія по замовчуванню рахує його дійсним? Чи є автоматична перевірка? А якщо ЦНАП помилився?
2. містобудівні умови  для Заліщицького району приходять на Чортків. 
3. Чортків погодив додаткове коло обробки земельниками. Чекаємо відповідальних.

**Comment by NizhnikAlex** [comment]
Извините, немного перепутал пользователей. Сейчас исправлено - все идет к своим пользователям. Для архитектуры создан тестовый аккаунт с логином/паролем Arkh_test

**Comment by kelnerira** [comment]
Так, дякуємо, все працює, тестуємо. Прохання - по замовчуванню робити для наших послуг однаковий формат обробки у ЦНАПі з додатковими полями, які друкуються у заяві: 
ПІБ посадової особи *  
  (поки відсутня можливість автоматично брати дані користувача) 
Дата оформлення документів у форматі - ДД.ММ.PPPP * 
  (поки відсутня можливість автоматично ставити дату обробки) 
Реєстраційний номер звернення
  (внутрішній порядковий номер у ЦНАП)

**Comment by NizhnikAlex** [comment]
Для земельників логін/пароль 
zem_test

**Comment by kelnerira** [comment]
По всіх земельних послугах, які є замовлені, актуальними є ті самі зауваження, що і по "Виправлення технічної помилки у відомостях з державного земельного кадастру, яка була допущена органом, що здійснює його ведення " щодо процесу опрацювання та друку документів, а саме:
 Дашборд чиновника:		
1	Зробити ще одне коло обробки для земельників	
2	результат послуги у земельників	 зробити можливість надання результату послуги без обов`язкового сканування документу (це може бути  5 листків, немає змісту сканувати бо людині потрібен оригінал)
3	Зробити кабінети для керівників з майбутніми ескалаціями на на e-mail, зараз - сповіщеннями про нові послуги,	Федущак Андрій Олегович  andrij8888@gmail.com
		Бурчик Ольга Іванівна ternopil.ru.te@land.gov.ua
4	друк заяви, змінити шапку, замість посади написати загальну назву органу	Відділу Держгеокадастру в Тернопільському районі
5	Друк заяви, забрати внизу підпис заявника	"Написати ""заява через електронний сервіс ternopil.igov.org.ua"", чи може така ""печатка"" як при заявці через ЕЦП? 
""Завірено iGov.org.ua
портал державних послуг"" тільки у нашому випадку щоб був регіональний портал"
6	1 коло обробки у ЦНАП, додати поля, які будуть друкуватися в заяві	Внутрішній номер
		Дата обробки
		ПІП Адміністратора ЦНАП

**Comment by kelnerira** [comment]
Перевірте також куди зникла динаміка - у обидвох послугах відсутні випадаючі поля, які є на основному порталі після вибору типу документа або виду змін, які потрібно внести.

**Comment by sofiadanylevych** [comment]
Отримали Технологічну та Інформаційну картки по екології, ті, які зараз відповідають процесу. Кажуть що послуга яку вони надають є згідно чинного законодавства і повинна надаватись так як у них, тому потрібно буде напевно що добавити ще коло для санепідем.
Технологічна картка 
https://docs.google.com/document/d/1higD3PIvTehFhHTaJ9YU-zDJ9IZYySFlluf_ZOmZgUA/edit?usp=sharing
Інформаційна картка 
https://docs.google.com/document/d/1eJaDEdqMem-_AxAxWrgYKs8CwgKtMyq5l582CX2McNc/edit?usp=sharing

**Comment by NizhnikAlex** [comment]
ок. Цікаво те, що в інфокарті прописаний лист СЕС, і більше нічого. А от в технологічній додані і документи для СЕС окремою припискою. Окремої послуги "Отримання листа СЕС" у вас не існує - перевірив на сайті. Спробую продавити в нас таку саме схему. Завтра додам ваш випадок для вашої області.

**Comment by sofiadanylevych** [comment]
Внесли зміни по екології,просимо подивитись
https://docs.google.com/spreadsheets/d/1It852JH-p7F6TFe5RwPIF6QPQf8CWBtjcPelxcGgfxs/edit?usp=sharing
